internal/queue/message/message_diagnostic.go

```diff
@@ -0,0 +1,10 @@
+package message
+
+// MessageDiagnostics contains all diagnostic data related to a single message.
+type MessageDiagnostics struct {
```

Comment:
Is it a good idea to create another struct other than the .proto auto generated Diagnostics struct duplicating it ?
I think it could be a better practice to use solely the .proto auto generated one

Reply:
This is used as the DTO, to be inserted in the database. It is not the same as the generated by the .proto (check the message.go for example, it contains other fields and also several struct tags)